x/ecocredit/basket/keeper/msg_update_curator_test.go

```diff
@@ -0,0 +1,112 @@
+package keeper
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/regen-network/gocuke"
+	"github.com/stretchr/testify/require"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	api "github.com/regen-network/regen-ledger/api/regen/ecocredit/basket/v1"
+	"github.com/regen-network/regen-ledger/types/testutil"
+	types "github.com/regen-network/regen-ledger/x/ecocredit/basket/types/v1"
+)
+
+type updateBasketCurator struct {
+	*baseSuite
+	alice sdk.AccAddress
+	bob   sdk.AccAddress
+	res   *types.MsgUpdateCuratorResponse
+	err   error
+}
+
+func TestUpdateBasketCurator(t *testing.T) {
+	gocuke.NewRunner(t, &updateBasketCurator{}).Path("./features/msg_update_curator.feature").Run()
+}
+
+func (s *updateBasketCurator) Before(t gocuke.TestingT) {
+	s.baseSuite = setupBase(t)
+	s.alice = s.addrs[0]
+	s.bob = s.addrs[1]
+}
+
+func (s *updateBasketCurator) ABasketWithPropertiesAndCuratorAlice(a gocuke.DocString) {
+	var basket types.Basket
+	err := json.Unmarshal([]byte(a.Content), &basket)
```

Comment:
```suggestion
	err := jsonpb.UnmarshalString(a.Content, &basket)
```

Reply:
We should be using `jsonpb` when we can. Will need to update the import as well.